.github/ISSUE_TEMPLATE/NEW_EXPERIMENT.md

```diff
@@ -0,0 +1,85 @@
+---
+name: "\U0001F4CB New experiment proposal"
+about: "Proposal and workflow guideline for adding a new experiment \U0001F52C."
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+<!-- ⚠️ If you do not respect this template, your issue will be closed -->
+<!-- ⚠️ Make sure to browse the opened and closed issues -->
+
+# New experiment proposal
+Provide the experiment and implementation details below and ask the qiskit-experiments core team to review 
```

Comment:
Most of the text within these sections are reminders for the author. I wonder if they would make more sense inside of `<!-- ... -->` tags so that they are visible to the author while writing the issue but the author does not need to delete all of them before submitting the issue.

Reply:
good idea, implemented